frontend/src/views/Auction/Auction.tsx

```diff
@@ -100,10 +98,14 @@ export const Auction = () => {
     }, [auctionData, profile]);
 
     useEffect(() => {
-        moreOptionsDropDownRef.current &&
-            (dropdownVisibility
-                ? (moreOptionsDropDownRef.current.style.opacity = "1")
-                : (moreOptionsDropDownRef.current.style.opacity = "0"));
+        if (moreOptionsDropDownRef.current && dropdownVisibility) {
```

Comment:
suggestion: you can probably toggle a class instead of setting the styles inline